File: C#/uri-1008.cs
// URI - 1008

using System;

namespace URI
{
    class Program
    {
        static void Main(string[] args)
        {
            int employeeNumber = int.Parse(Console.ReadLine());
            int workedHours = int.Parse(Console.ReadLine());
            double hourlyValue = double.Parse(Console.ReadLine());

            Console.WriteLine("NUMBER = " + employeeNumber);
            Console.WriteLine("SALARY = U$ " + (workedHours * hourlyValue).ToString("F2"));
        }
    }
}

File: Ruby/uri-1008.rb
#URI - 1008: https://www.urionlinejudge.com.br/judge/pt/problems/view/1008

Employee_number = gets.to_i
Number_of_hours_worked = gets.to_i
Value_per_hour = gets.to_f
Salary = Number_of_hours_worked * Value_per_hour
puts "NUMBER = #{Employee_number}"
puts "SALARY = U$ #{'%.2f' % Salary}"